microsoft/botframework-components
# Conversational core template using Preview.Bot packages

### Purpose
*What is the context of this pull request? Why is it being done?*
Updates #496 

### Changes
*Are there any changes that need to be called out as significant or particularly difficult to grasp? (Include illustrative screenshots for context if applicable.)*
- Template using HelpAndCancel, Welcome packages

### How to test
*Is this covered by existing tests or new ones? If no, why not?*
1. Pull down this branch.
2. Navigate to \generators\generator-conversational-core
3. Run `npm install`
4. Navigate to another directory to create your own bot and run `yo {path_to_\generators\generator-conversational-core\app\index.js {bot-name}`
5. Template should generate successfully and you can open in Composer to validate usability

### Feature Plan
*Are there any remaining steps or dependencies before this issue can be fully resolved? If so, describe and link to any relevant pull requests or issues.*
- Preview packages need to be renamed and published, this template will need to be updated then
